sklearn/feature_selection/_from_model.py

```diff
@@ -97,9 +97,9 @@ class SelectFromModel(MetaEstimatorMixin, SelectorMixin, BaseEstimator):
 
     threshold : str or float, default=None
```

Comment:
```suggestion
    threshold : {"mean", "median"} or float, default=None
```

Reply:
Uhm actually it is more than these 2 options since we can pass a factor. You can discard this suggestion.